### PR TITLE
Updating TOB from 2022 election

### LINKED ISSUE #119: TOB listing needs to be updated
The [website](https://opencontainers.org/about/tob/) needs to be updated with the results of the election (see #113). The relevant page is at:

https://github.com/opencontainers/opencontainers.org/blob/master/content/about/tob.md

### PR DESCRIPTION
**Pull request by sudo-bmitch**
Signed-off-by: Brandon Mitchell <git@bmitch.net>

Better late than never. For the new(er) TOB members, feel free to suggest descriptions and profile photos (or send a separate PR if this gets merged). @tych0 @jdolitsky @nishakm 

Fixes #119

Related to #113

### CODE REVIEW

File: content/about/tob.md [hunk 1]
```diff
@@ -39,21 +39,23 @@ The TOB holds infrequent meetings which are usually recorded and posted on [the
...
 as the Go compiler for Fedora and Red Hat.
-
 {{< /card >}}
 
-{{< card img="/img/tob/Michael-Crosby.jpeg" name="Michael Crosby" termdates="2019-01-29 → 2021-01-29" company="Apple" github="https://github.com/crosbymichael" twitter="https://twitter.com/crosbymichael" >}}
+{{< card name="Josh Dolitsky" termdates="2022-01-29 → 2024-01-29" company="Chainguard" github="https://github.com/jdolitsky" >}}
 {{< /card >}}
 
-{{< card img="/img/tob/Phil-Estes.jpg" name="Phil Estes" termdates="2020-01-29 → 2022-01-29" title="TOB Chair" company="AWS" github="https://github.com/estesp" twitter="https://twitter.com/estesp" >}}
+{{< card img="/img/tob/Phil-Estes.jpg" name="Phil Estes" termdates="2022-01-29 → 2024-01-29" title="TOB Chair" company="AWS" github="https://github.com/estesp" twitter="https://twitter.com/estesp" >}}
```
Comment (samuelkarp) [inline content/about/tob.md:58]: Phil isn't the chair anymore.

File: content/about/tob.md [hunk 2]
```diff
@@ -62,29 +64,25 @@ CNCF containerd project and has been active in the OCI since its founding
...
 Software engineer at Google working on containers.
 {{< /card >}}
 
-{{< card img="/img/tob/Samuel-Karp.jpeg" name="Samuel Karp" termdates="2020-01-29 → 2022-01-29" company="Amazon Web Services" github="https://github.com/samuelkarp" twitter="https://twitter.com/samuelkarp" personal="https://samuelkarp.com" >}}
+{{< card img="/img/tob/Samuel-Karp.jpeg" name="Samuel Karp" termdates="2022-01-29 → 2024-01-29" company="Amazon Web Services" github="https://github.com/samuelkarp" twitter="https://twitter.com/samuelkarp" personal="https://samuelkarp.com" >}}
 Samuel Karp is a Senior Software Development Engineer working on containers at
 Amazon Web Services. Sam first contributed to Docker in 2015 and containerd in
 2017. Sam's current focus is on container runtimes and how the runtimes
 interact with orchestrators and registries. Sam has written code and designs
 for Amazon ECS and AWS Fargate and consulted for Amazon ECR and Amazon EKS.
```
Comment (samuelkarp) [inline content/about/tob.md:76]: My company affiliation has changed (Google) and the description isn't super accurate anymore.  Do you prefer to get this merged first and then have TOB members update their own description or would you prefer to have it as a suggestion here?

Comment (sudo-bmitch) [reply content/about/tob.md]: Either is fine with me. While I've got the PR open, feel free to send the change over to me.

File: content/about/tob.md (hunk 1 above)
Comment (sudo-bmitch) [reply content/about/tob.md]: Fixed.

File: content/about/tob.md [hunk 3]
```diff
@@ -39,21 +39,23 @@ The TOB holds infrequent meetings which are usually recorded and posted on [the
 
 {{< card-container >}}
 
-{{< card img="/img/tob/Vincent-Batts.jpeg" name="Vincent Batts" termdates="2019-01-29 → 2021-01-29" company="Kinvolk" github="https://github.com/vbatts" linkedin="https://www.linkedin.com/in/vincentbatts/" twitter="https://twitter.com/vbatts" >}}
+{{< card img="/img/tob/Tycho-Andersen.jpeg" name="Tycho Andersen" termdates="2021-01-29 → 2023-01-29" company="Cisco" github="https://github.com/tych0" >}}
```
Comment (tych0) [inline content/about/tob.md:42]:
```suggestion
{{< card img="/img/tob/Tycho-Andersen.jpeg" name="Tycho Andersen" termdates="2021-01-29 → 2023-01-29" company="Netflix" github="https://github.com/tych0" >}}
```

Other than that lgtm.

Comment (sudo-bmitch) [reply content/about/tob.md]: Got it, thanks!

File: content/about/tob.md (hunk 2 above)
Comment (sudo-bmitch) [reply content/about/tob.md]: @samuelkarp send over your updated description since the affiliation change got included in my last update. Thanks!

Comment (sudo-bmitch) [reply content/about/tob.md]: Updated. Thanks!

**Comment by sudo-bmitch** [comment]
@caniszczyk I think this is good enough to merge now. We can add other descriptions and photos with separate PRs.

Preview is available at: https://deploy-preview-124--opencontainers.netlify.app/about/tob/

**Review by samuelkarp — APPROVED** [review]

**Pull request merged**